cms/static/sass/_build-v1.scss

```diff
@@ -10,7 +10,7 @@
 
 // +Base - Utilities
 // ====================
-@import 'variables';
+@import 'partials/variables';
```

Comment:
Note: this was just a pre-existing typo. When I started using variables in _modules.scss, Lettuce tests failed due to "variables" not being properly included. I'm not sure why no failures occurred before, except that "variables" was only used in a couple of places with CMS (so perhaps no Lettuce tests hit the related files).